src/app/container/refresh-wizard/refresh-wizard.component.html

```diff
@@ -0,0 +1,72 @@
+<div class="modal-body">
+  <p>
+    Select a Quay.io namespace and then click the sync icon next to a repository to automatically register or sync the tool on Dockstore.
+    Requires a Quay.io account linked on Dockstore and also Git build triggers to be present on the repository.
+  </p>
+  <app-loading [loading]="loading$ | async">
+    <mat-card class="alert alert-danger" role="alert" *ngIf="error$ | async as error">
+      <button type="button" class="close" data-dismiss="alert" (click)="clearError()">
+        &times;
+      </button>
+      <p>
+        <mat-icon>warning</mat-icon>
+        The webservice encountered an error.
+      </p>
+      <p class="error-output">
+        {{
+          '[HTTP ' +
+            error.status +
+            '] ' +
+            error.statusText +
+            ': ' +
+            (error.error && error.error.message ? error.error.message : error.error)
```

Comment:
If `error.error` is falsey, e.g., null, then this will display `...: null`, if it's a concern.

Reply:
I know we do this elsewhere, but I feel like we shouldn't display the error code to users, and instead just use an error message.

Reply:
This is actually copied from the refresh component (because I don't think it's worth warranting creating another shared component). The error code discussion has been brought up before, but I think Denis wanted it there?

Reply:
Yeah, I actually like displaying more information about errors than less. 
I feel it gives more technical users a chance at understanding the problem, ourselves a quicker source of information (what error code did you get?), and there's not much cost aside from less savvy users seeing a more technical error message.